Dshop/apps/products_catalogue/api_urls.py

```diff
@@ -0,0 +1,10 @@
+from django.urls import path, include
+from .api_views import ProductViewSet
+from rest_framework.routers import DefaultRouter
+
+router = DefaultRouter()
```

Comment:
I think it would be a good idea to remove the trailing slashes. My front-end friend recently pointed out that it was easier for them that way.
`router = DefaultRouter(trailing_slash=False)`

Reply:
I dont have preference @knop-k. Your call.